uniffi_bindgen/src/bindings/swift/templates/BridgingHeaderTemplate.h

```diff
@@ -48,7 +43,7 @@ typedef struct RustCallStatus {
 // ⚠️ Attention: If you change this #else block (ending in `#endif // def UNIFFI_SHARED_H`) you *must* ⚠️
 // ⚠️ increment the version suffix in all instances of UNIFFI_SHARED_HEADER_V2 in this file.           ⚠️
 #endif // def UNIFFI_SHARED_H
```

Comment:
Hrm...I wonder if technically we should be revving to `UNIFFI_SHARED_HEADER_V2` here. Otherwise, if someone accidentally compiles together two UniFFIed crates that use different versions of UniFFI, one of them will end up using a `RustBuffer` struct definition with missing (or extra) fields. @badboy what do you think?

(I'm also curious if the warnings in the code here could have been better, to help better understand whether we need to do this)

Reply:
I just didn't _read_ that file, I did a simple search-and-replace for the padding fields.
Now re-reading it I agree, a version increment is necessary now